scilpy/reconst/multi_processes.py

```diff
@@ -446,6 +447,16 @@ def convert_sh_basis(shm_coeff, sphere, mask=None,
     nbr_processes: int, optional
         The number of subprocesses to use.
         Default: multiprocessing.cpu_count()
+    is_input_legacy: bool, optional
+        If true, this means that the input SH used a legacy basis definition
+        for backward compatibility with previous ``tournier07`` and
+        ``descoteaux07`` implementations.
+        Default: True
+    is_output_legacy: bool, optional
+        If true, this means that the output SH will use a legacy basis
+        definition for backward compatibility with previous ``tournier07`` and
+        ``descoteaux07`` implementations.
```

Comment:
If there is a link to mrtrix or dipy documentation about that. It would be useful here and/or in the script